.github/workflows/build.yml

```diff
@@ -41,8 +41,8 @@ jobs:
           push: true
           tags: ${{ steps.login-ecr-vaec.outputs.registry }}/notification_api:${{ inputs.ref }}_${{ inputs.environment }}
 
-      - name: Dispatch Twistlock Workflow
-        env:
-          GH_TOKEN: ${{ github.token }}
-        run: |
-          gh workflow run -r master -F image-tag=${{ inputs.ref }}_${{ inputs.environment }} twistlock.yml
+      #- name: Dispatch Twistlock Workflow
```

Comment:
Please add a comment as to why this is disabled.

Reply:
Add the ticket number 1398 to that comment, please